src/tooltip/tooltip.component.ts

```diff
@@ -99,4 +102,14 @@ export class Tooltip extends PopoverContainer {
 	public isTemplate(value) {
 		return value instanceof TemplateRef;
 	}
+
+	ngAfterViewInit() {
+		if (this.wrapper) {
+			// Buttons
+			const buttonElement = this.wrapper.nativeElement.querySelector("button");
+			if (buttonElement && !buttonElement.getAttribute("aria-labelledby")) {
+				buttonElement.setAttribute("aria-labelledby", this.id);
+			}
+		}
+	}
```

Comment:
I'm doing some magic here, there is an easier way but it would require users to do the following:

```html
<!-- Setting `#tooltip` as a template reference variable to the component -->
<ibm-tooltip
			#tooltip
			[isOpen]="isOpen"
			[caret]="caret"
			[align]="align"
			(onOpen)="onOpen($event)"
			(onClose)="onClose($event)"
			(isOpenChange)="isOpenChange($event)"
			[description]="description">
			<!-- Retrieving the id & assigning it to the button -->
			<button type="button" class="tooltip-trigger" [attr.aria-labelledby]="tooltip.id">
				<svg preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 32 32">
					<path d="M26,4H6A2,2,0,0,0,4,6V26a2,2,0,0,0,2,2H26a2,2,0,0,0,2-2V6A2,2,0,0,0,26,4ZM6,26V6H26V26Z"></path>
				</svg>
			</button>
</ibm-tooltip>
```